src/annotator/components/ContentInfoBanner.js

```diff
@@ -8,39 +15,108 @@ import { LabeledButton, Link } from '@hypothesis/frontend-shared';
  * @param {object} props
  *   @param {ContentInfoConfig} props.info
- *   @param {() => void} props.onClose
  */
-export default function ContentInfoBanner({ info, onClose }) {
+export default function ContentInfoBanner({ info }) {
   return (
-    <div className="flex items-center border-b gap-x-4 px-2 py-1 bg-white text-annotator-lg">
-      <Link href={info.logo.link} target="_blank" data-testid="logo-link">
-        <img src={info.logo.logo} alt={info.logo.title} />
-      </Link>
-      <div className="grow">
-        {info.links.previousItem && (
-          <Link href={info.links.previousItem} target="_blank">
-            Previous article
+    <div
+      className={classnames(
+        'h-10 bg-white px-4 text-slate-7 text-annotator-base border-b',
+        'grid items-center',
+        // Two columns in narrower viewports; three in wider
+        'grid-cols-[100px_minmax(0,auto)] gap-x-4',
+        '2xl:grid-cols-[100px_minmax(0,auto)_minmax(0,auto)] 2xl:gap-x-3'
+      )}
+    >
+      <div data-testid="content-logo">
+        {info.logo && (
+          <Link href="https://jstor.org" target="_blank">
+            <img alt={info.logo.title} src={info.logo.logo} />
           </Link>
         )}
-        {info.item.title}
-        {info.item.containerTitle && (
-          <span>
-            {' '}
-            from <i>{info.item.containerTitle}</i>
-          </span>
-        )}
-        {info.links.nextItem && (
-          <Link href={info.links.nextItem} target="_blank">
-            Next article
-          </Link>
+      </div>
+      <div
+        className={classnames(
+          // Container title (this element) is not shown on narrow screens
+          'hidden',
+          '2xl:block 2xl:whitespace-nowrap 2xl:overflow-hidden 2xl:text-ellipsis'
         )}
+        data-testid="content-container-info"
+      >
+        <span className="font-semibold" title={info.item.containerTitle}>
+          {info.item.containerTitle}
+        </span>
       </div>
-      <div className="text-annotator-base">
-        <LabeledButton onClick={onClose} data-testid="close-button">
-          Close
-        </LabeledButton>
+      <div
+        className={classnames(
+          // Flex layout for item title, next and previous links
+          'flex justify-center items-center gap-x-2'
+        )}
+        data-testid="content-item-info"
+      >
+        <div
+          className={classnames(
+            // Narrower viewports center this flex content:
+            // this element is not needed for alignment
+            'hidden',
+            // Wider viewports align this flex content to the right:
+            // This empty element is needed to fill extra space at left
+            '2xl:block 2xl:grow'
+          )}
+        />
+        {info.links?.previousItem && (
+          <>
+            <Link
+              classes="flex gap-x-1 items-center text-annotator-sm whitespace-nowrap"
+              title={`Open previous item in a new window`}
```

Comment:
I don't think we need the "in a new window" suffix. In fact I would avoid using a title at all, since the label will be more specific (eg. "Previous chapter", "Previous article"). If we put `target="_blank"` on the link then the browser can provide whatever hints it normally provides to indicate that the link will open in a new tab.

Reply:
@robertknight  Oh, note, we don't currently have data for what type of content it is. So if we want to add "chapter" or "article", we'll need that.

Reply:
Note that I am still keeping a `title` here because we need some way to show the full title of a potentially-truncated item on hover (disclosure). But I am removing the other language.